GitHub Actions fail on macos

## Description of the bug

GitHub Actions fail on macos.
This issue has been found during #456.
Here are my findinds:

- The macOS environment got updated from `20200610.3` to `20200618.1` 6/24 6:13PM (PDT) (https://github.com/actions/virtual-environments/commit/15c14c65f0e07a6664d6eb9624bb8827d2576b73#diff-e9f9822df5b3b9087dd94d4c3c083277)
- The last successful run on the master branch was 6/23 6:59PM (PDT) and was using `20200610.3` (https://github.com/HazyResearch/fonduer/actions/runs/145616087).
- Recent runs failed and they were using `20200618.1`.

## To Reproduce
Steps to reproduce the behavior:
1. Run GitHub Actions on the master branch

## Expected behavior

GitHub Actions succeed on macos.

## Error Logs/Screenshots

https://github.com/HiromuHota/fonduer/actions/runs/151774864
```
...


==> Installing imagemagick
==> Pouring imagemagick-7.0.10-22.catalina.bottle.tar.gz
🍺  /usr/local/Cellar/imagemagick/7.0.10-22: 1,497 files, 25.6MB
==> Caveats
==> docbook
To use the DocBook package in your XML toolchain,
you need to add the following to your ~/.bashrc:

export XML_CATALOG_FILES="/usr/local/etc/xml/catalog"
==> gnu-getopt
gnu-getopt is keg-only, which means it was not symlinked into /usr/local,
because macOS already provides this software and installing another version in
parallel can cause all kinds of trouble.

If you need to have gnu-getopt first in your PATH run:
  echo 'export PATH="/usr/local/opt/gnu-getopt/bin:$PATH"' >> /Users/runner/.bash_profile


Bash completion has been installed to:
  /usr/local/opt/gnu-getopt/etc/bash_completion.d
==> libtool
In order to prevent conflicts with Apple's own libtool we have prepended a "g"
so, you have instead: glibtool and glibtoolize.
##[error]Process completed with exit code 1.
```

## Environment (please complete the following information)
 - OS: macos on GitHub Actions (20200618.1)

## Additional context

The difference between `20200610.3` and `20200618.1`: https://github.com/actions/virtual-environments/compare/macos-10.15/20200610.3...macos-10.15/20200618.1

--- a/.github/workflows/pythonpackage.yml
+++ b/.github/workflows/pythonpackage.yml
@@ -93,13 +93,11 @@ jobs:
         command: |
           pip install -U pip  # upgrade pip after installing python
           echo "::add-path::`pyenv root`/shims"
-    - name: Setup tmate session
-      uses: mxschmitt/action-tmate@v2
     - name: Install dependencies for macOS
       run: |
         brew update
         # Install ImageMagick for Wand.
-        brew install freetype imagemagick ghostscript
+        brew install imagemagick
 
         brew install poppler
 

--- a/.github/workflows/pythonpackage.yml
+++ b/.github/workflows/pythonpackage.yml
@@ -93,6 +93,8 @@ jobs:
         command: |
           pip install -U pip  # upgrade pip after installing python
           echo "::add-path::`pyenv root`/shims"
+    - name: Setup tmate session
+      uses: mxschmitt/action-tmate@v2
     - name: Install dependencies for macOS
       run: |
         brew update

